src/Mutator/Arithmetic/Multiplication.php

```diff
@@ -94,7 +96,19 @@ public function canMutate(Node $node): bool
             return false;
         }
 
-        return true;
+        $functionScope = $node->getAttribute(ReflectionVisitor::FUNCTION_SCOPE_KEY);
```

Comment:
I believe we should also check that the current node is a `return` expression as it's done e.g. here: https://github.com/infection/infection/blob/master/src/Mutator/ReturnValue/This.php#L84

Otherwise, the following code will not be mutated, but it should:

```php
public function mul(int $a, int $b): int
{
    $c = $a * $b;

    return 1;
}
```

btw, could you please add such case to tests so that we are sure it is mutated?

Reply:
done